Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/local/source/LocalStatisticsCache.kt

```diff
@@ -0,0 +1,44 @@
+package de.rki.coronawarnapp.statistics.local.source
+
+import dagger.assisted.Assisted
+import dagger.assisted.AssistedFactory
+import dagger.assisted.AssistedInject
+import de.rki.coronawarnapp.statistics.Statistics
+import de.rki.coronawarnapp.statistics.local.FederalStateToPackageId
+import timber.log.Timber
+import java.io.File
+
+class LocalStatisticsCache @AssistedInject constructor(
+    @Statistics cacheDir: File,
+    @Assisted private val stateForCache: FederalStateToPackageId,
+) {
```

Comment:
Why use one instance per federal state, instead of storing all of the states within one instance and a method that takes the ID as parameter?